logrotate.8.in

```diff
@@ -705,6 +708,11 @@ Scripts are run as the invoking user and group, irrespective of any \fBsu\fR
 directive.  If the \fB\-\-log\fR flag was specified, file descriptor 3 is the
 log file.
 
+.SH USER AND GROUP
+
+User and group identifiers are resolved first by trying the textual
+representation and only afterwards by the numeric value.
```

Comment:
Instead of `... and only afterwards ...` I would say `... and, in case it fails, ....`.  Otherwise one could understand it as we always do two-steps resolution or something.

Reply:
reworded